src/nitpackage.nit

```diff
@@ -74,6 +82,49 @@ private class ReadmePhase
 		var path = mpackage.expand
 		toolcontext.info("{mpackage} moved to {path}", 0)
 	end
+
+	fun gen_ini(mpackage: MPackage) do
+		if not toolcontext.opt_gen_ini.value then return
+		if mpackage.has_ini and not toolcontext.opt_upd_ini.value then return
+
+		var ini_path = mpackage.ini_path
+		if ini_path == null then return
+		var has_ini = mpackage.has_ini
+
+		var ini = new ConfigTree(ini_path)
+
+		update_ini_value(ini, "package.name", mpackage.name)
+		update_ini_value(ini, "package.desc", "")
```

Comment:
The current spec read the MPackage synopsis from the README, but I want to gen the README from other data since in most case the README starts with `# MyPackage`.

I would prefer to read the synopsis of a MPackage from the ini.

Reply:
I think it's OK to support both way, ie it one is empty it can fill the other